### DISCUSSION TITLE
Contribute to the documentation

### DISCUSSION THREAD

**Discussion by truonghm** [Nixtla/mlforecast#167] [Q&A]
I found a small [formatting issue](https://nixtla.github.io/mlforecast/core.html) in the documentation of `mlforecast` and would like to make a PR to fix this issue. However I'm not sure how to build the docs locally to see if the issue has been updated by my fix. 

I have looked through the codebase and haven't found any instruction on how to build just the docs only. Is there anyway to do this? 

Thank you!

![image](https://github.com/Nixtla/mlforecast/assets/10404416/925c759a-3b21-4c9b-a48e-ef0d9ddfc5f7)

**Comment by jmoralez** [comment]
Sure. You just need to make the change in the notebook, save it and then run nbdev_preview in a terminal. You need to have nbdev installed and that may install quarto, but after running successfully it'll open a browser tab where you'll be able to see them.
Please let us know if you run into any issues.
